static/riotjs/wizard/create_team.tag

```diff
@@ -14,17 +14,19 @@
         <div class="fields">
             <div class="eight wide field">
                 <label>Members:</label>
```

Comment:
Changed these member fields to multiple select Semantic UI dropdowns.

Reply:
👍